static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -21,33 +21,29 @@ define([
 ) {
     function vote(event, signedIn) {
         var component = event.currentTarget.closest('.js-view-tracking-component');
+        var votes = $('.user__question-container input:checked', component);
 
-        var votes = Array.from(component.querySelectorAll('.user__question-container')).filter(function(question) {
-            return $('input', question).attr('checked')
-        });
+        if (votes) {
```

Comment:
`$` returns an array, so this will always be true. instead, use `if(votes.length)`